replica/table.cc

```diff
@@ -2787,7 +2792,6 @@ future<> tablet_storage_group_manager::merge_completion_fiber() {
     while (!_t.async_gate().is_closed()) {
         try {
             co_await utils::get_local_injector().inject("merge_completion_fiber", utils::wait_for_message(60s));
-
```

Comment:
nit: please do not mix cosmetic (and unneeded) changes with functional changes

Reply:
accident